requirements.txt

```diff
@@ -1,4 +1,5 @@
 awscli==1.16.155
 boto3==1.9.145
 troposphere==2.4.6
-pytest==4.5.0
\ No newline at end of file
+pytest==4.5.0
+pylint
```

Comment:
Does this not need a version?

Reply:
I left the version of pylint out on purpose. My thinking was the following:
* version control tightly all the dependencies required to provide the service
* because it is not used in the code itself, don't specify a version of the linter so that the latest version is used

Reply:
Okay. Sounds like a sensible approach.